pkg/testing/integration/program.go

```diff
@@ -2242,14 +2243,13 @@ func (pt *ProgramTester) prepareGoProject(projinfo *engine.Projinfo) error {
 	}
 
 	// link local dependencies
-	for _, pkg := range pt.opts.Dependencies {
-		var editStr string
-		if strings.ContainsRune(pkg, '=') {
-			// Use a literal replacement path.
-			editStr = pkg
-		} else {
-			dep := getRewritePath(pkg, gopath, depRoot)
-			editStr = fmt.Sprintf("%s=%s", pkg, dep)
+	for _, dep := range pt.opts.Dependencies {
+		// when we load a package from a relative dir string, we read the package name from go.mod
+		// Use a literal replacement path.
+		/* pkg is a directory path, create a replace edit string from it */
```

Comment:
Nit: Why the different comment style?

Reply:
Deleted, this comment doesn't make sense after the refactoring.